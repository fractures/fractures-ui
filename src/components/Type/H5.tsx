import PropTypes from 'prop-types'
import styled from 'styled-components'

const propTypes = {
	children: PropTypes.node.isRequired,
	className: PropTypes.string
}

const H5 = styled.h5`
	margin-bottom: 0;
	margin-top: 0;

	font-size: 1.125rem;
	line-height: 1.375rem;
`

H5.propTypes = propTypes

export default H5
