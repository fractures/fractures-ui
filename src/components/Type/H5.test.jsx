import H5 from './H5'

describe('H5', () => {
	const tree = renderer.create(<H5>Extra header</H5>).toJSON()

	it('Matches snapshot', () => {
		expect(tree).toMatchSnapshot()
	})
})
