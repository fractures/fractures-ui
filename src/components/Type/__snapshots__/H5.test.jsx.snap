// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`H5 Matches snapshot 1`] = `
<h5
  className="sc-bdVaJa bxtxFX"
>
  Extra header
</h5>
`;
